### PR TITLE
feat: post reopened issues to channel

### PR DESCRIPTION
**Pull request by bo0tzz**

### CODE REVIEW

File: src/services/webhook.service.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ export class WebhookService {
       return;
     }
 
-    if ('issue' in dto && (action === 'opened' || action === 'closed')) {
+    if ('issue' in dto && (action === 'opened' || action == 'reopened' || action === 'closed')) {
```
Comment (bo0tzz) [inline src/services/webhook.service.ts:75]: Could these be expressed as `if ('issue' in dto && action in ['opened', 'reopened', 'closed'])`? That seems clearer to me (esp for the longer PR match above).

Comment (danieldietzler) [reply src/services/webhook.service.ts]: Nope, wanted to do that as well but then typescript does not understand the type anymore :(

**Review by jrasm91 — APPROVED** [review]

File: src/services/webhook.service.ts (hunk 1 above)
Comment (bo0tzz) [reply src/services/webhook.service.ts]: lame

Comment (jrasm91) [reply src/services/webhook.service.ts]: You probably can, actually. But you might need to add `[...] as const` or another type

Comment (danieldietzler) [reply src/services/webhook.service.ts]: Oh `as const` may make it work, true!

**Pull request merged**